zim/plugins/toolbar.py

```diff
@@ -36,6 +36,8 @@
 
 import zim.errors
 
+import os  # see issue #2007 - Gtk bug on other platforms
+DEFAULT_DECOR = os.uname().sysname == 'Linux'
```

Comment:
`AttributeError: module 'os' has no attribute 'uname'` on Windows. Since this isn't a one-and-done, we might as well account for the BSDs - they likely support the same stuff as Linux via X11, so the workaround needs not apply.